cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -7,78 +7,67 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [id="odo-notable-improvements_{context}"]
-== Notable changes and improvements in `{odo-title}`
-
-* `{odo-title}` now supports Devfile v2.
-
-* `odo create -s2i` now converts an S2I component into a devfile component. When running `odo create --s2i <component-type>` `{odo-title}` now creates a converted Devfile component based on the S2I images of the specified component type.
+== Notable changes and improvements in `{odo-title}` version 2.5.0
+
+// #5238
+* Creates unique routes for each component, using `adler32` hashing
+// #5252
+* Supports additional fields in the devfile for assigning resources:
+** cpuRequest
+** cpuLimit
+** memoryRequest
+** memoryLimit
+// #5276
+* You can now run the `odo delete` command with the `--deploy` flag to remove components that you deployed using the `odo deploy` command: 
```

Comment:
Since all the other features start with verbs, suggest something on the lines of: Adds the `--deploy` flag to the `odo delete` command to remove components deployed using the `odo deploy` command: